scripts/generate-cni-yaml.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+PLATFORM=$(uname | tr '[:upper:]' '[:lower:]')
+HELM_VERSION="3.0.2"
+NAMESPACE="kube-system"
+
+MAKEFILEPATH=$SCRIPTPATH/../Makefile
+VERSION=$(make -s -f $MAKEFILEPATH version)
+BUILD_DIR=$SCRIPTPATH/../build/cni-rel-yamls/$VERSION
```

Comment:
make sure `build/` is added to your `.gitignore`

Reply:
Right, will do that.